theory-test-admission-candidate-app/app/server/services/candidateCheckBookingService.js

```diff
@@ -1,21 +1,61 @@
 const AWS = require('aws-sdk');
+const moment = require('moment');
+const logger = require('logger');
+
+const bookingStepFunction = new AWS.StepFunctions();
```

Comment:
bad name - this isn't _your_ step function, it's an AWS Service Object that lets you interact with _all_ step functions

Reply:
What would you suggest?